gulpfile.js

```diff
@@ -183,6 +184,7 @@ gulp.task('vulcanize-and-minify', function() {
       stripExcludes: false,
       excludes: [path.resolve('./dist/third_party/polymer.html')]
     }))
+    .pipe(base64())
```

Comment:
gulp-base64 task is for converting all files found within a url( ... ) declaration into base64-encoded data URI strings. Since `tick.svg` and `up-down.svg` are declared in the html, I thought this was the place to make sure they were well encoded.